pkg/cronjob/es_index_cleaner.go

```diff
@@ -91,6 +92,21 @@ func CreateEsIndexCleaner(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 							},
 							RestartPolicy: corev1.RestartPolicyNever,
 							Volumes:       jaeger.Spec.Volumes,
+=======
```

Comment:
Looks like this is the remnant of a merge conflict :-)

Reply:
Ahh my mistake! will fix